src/main/java/core/basesyntax/data/ReportService.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.data;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class ReportService {

```

Comment:
do not violate the principle of Dependency Inversion. You should implement interface

Reply:
There is still no interface for this class